### PR TITLE
#1202 Initial add About page and added to header nav.

### LINKED ISSUE #1202: v2 Add About Page
### Overview
We need to add an about page to the v2 site to provide background and context of the 311-Data project. 

### Action Items
- [x] Add page 
- [ ] Use design panel in Figma as content 
- [ ] Ensure all hyperlinks are used 
- [x] Add "About Us" to header navigation 

### Resources/Instructions
Design work was completed [here](https://github.com/hackforla/311-data/issues/960).

Page design found in [Figma Design Studio](https://www.figma.com/file/eF63O8yIGz2soQ4JJFfyoK/311_Data_UXUI?node-id=2775%3A7702) under Sim -> farthest right. 
Snip20220513_36

### LINKED ISSUE #1264: Make all request types selected by default
In PR #1149, we changed all the request types to be unselected by default. I think that is a bit unintuitive--when a user first visits the site, they will not see any requests, and then they will have to check some types to see them. I think it's more user friendly to start with seeing all the requests, and then the user can filter from there if they'd like.

We'd want to change all [these](https://github.com/hackforla/311-data/blob/dev/client/redux/reducers/filters.js#L41) values to true. I believe this would select all of the request types on the UI, but the "Select All" button wouldn't be checked, so we'd need to do additional work for that.

### PR DESCRIPTION
**Pull request by funbunch**
Fixes #1202

 Current view: 
Screen Shot 2022-07-02 at 2 41 44 PM

  - [ ] Up to date with `dev` branch
  - [ ] Branch name follows [guidelines](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md#feature-branching)
  - [ ] All PR Status checks are successful
  - [ ] Peer reviewed and approved

Any questions? See the [getting started guide](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md)

### CODE REVIEW

File: client/components/Header.jsx [hunk 1]
```diff
@@ -110,8 +110,8 @@ const Header = () => {
         <NavLink to="/faqs" className={classes.link} activeStyle={activeStyle}>
           <Button className={classes.button}>FAQ</Button>
         </NavLink>
-        <NavLink to="/blog" className={classes.link} activeStyle={activeStyle}>
```
Comment (nichhk) [inline client/components/Header.jsx:113]: Now that I'm thinking about it, we might want to keep the blog. So please just add a new NavLink for About, and keep the existing NavLink for Blog.

**Review by nichhk — CHANGES_REQUESTED** [review]
Thanks Shannan! Looks good. One small comment.

**Comment by nichhk** [comment]
Also, the original comment is incorrect--this PR contributes to #1202.

File: client/components/Header.jsx (hunk 1 above)
Comment (funbunch) [reply client/components/Header.jsx:113]: Will do.

**Comment by nichhk** [comment]
Looks good! Thanks Shannan. Please merge the PR yourself when you're ready.

**Review by nichhk — APPROVED** [review]

**Pull request merged**